MLendPoint/src/middleware/obtainDirectory.js

```diff
@@ -1,12 +1,12 @@
 const fs = require("fs");
 const path = require("path");
 
-class ObtainDirectory{
-    constructor() {}
+class ObtainDirectory {
+    constructor() { }
```

Comment:
This constructor don't have any arguments. You should remove  this constructor if you don't use it